internal/db/collection_get.go

```diff
@@ -59,11 +60,10 @@ func (c *collection) get(
 	showDeleted bool,
 ) (*client.Document, error) {
 	txn := mustGetContextTxn(ctx)
-	identity := GetContextIdentity(ctx)
 	// create a new document fetcher
 	df := c.newFetcher()
 	// initialize it with the primary index
-	err := df.Init(ctx, identity, txn, c.db.acp, c, fields, nil, nil, false, showDeleted)
+	err := df.Init(ctx, identity.FromContext(ctx), txn, c.db.acp, c, fields, nil, nil, false, showDeleted)
```

Comment:
question: What made you do this?

Reply:
if you asked why I move it to `identity` package? So that other package don't depend on `db` just to get an identity from a `Context`

Reply:
gotchu, thanks